src/MessageContainer.js

```diff
@@ -135,10 +135,20 @@ export default class MessageContainer extends React.Component {
     );
   }
 
+  endReachedProps(){
+    let condProps = {}
+    if (this.props.onTopReached){ condProps.onEndReached = this.props.onTopReached }
```

Comment:
In my opinion it seems a good idea.

Anyway, in case that your PR will be accepted there, you need to make a new one here to allow pass the function and threshold through `GiftedChat` (maybe arround [here](https://github.com/FaridSafi/react-native-gifted-chat/blob/91312c16f041bd78c61d98d0cbdf3f539a7f0f24/src/GiftedChat.js#L68)?) and add some documentation about the `MessageContainer` component uses a [`InvertibleScrollView`](https://github.com/exponentjs/react-native-invertible-scroll-view), explaining that actually _onTopReached_ and _onTopReachedThreshold_ are only alias of _onEndReached_ and _onEndReachedThreshold_ in order to improve the comprehension.

Isn't it right?